editoast/src/main.rs

```diff
@@ -99,6 +104,7 @@ async fn runserver(
             .app_data(Data::new(CHashMap::<i64, InfraCache>::default()))
             .app_data(Data::new(MapLayers::parse()))
             .app_data(Data::new(args.map_layers_config.clone()))
+            .app_data(Data::new(search_config.clone()))
```

Comment:
```suggestion
            .app_data(Data::new(SearchConfig::parse()))
```
to be consistent with `MapLayers`